util/argo/resource_tracking.go

```diff
@@ -170,9 +170,17 @@ func (rt *resourceTracking) Normalize(config, live *unstructured.Unstructured, l
 		return err
 	}
 
-	err = argokube.SetAppInstanceLabel(config, labelKey, label)
-	if err != nil {
-		return err
+	labels := live.GetLabels()
```

Comment:
I think it will be better move this part to argokube module

Reply:
Done.